prisma/schema.prisma

```diff
@@ -49,10 +52,14 @@ model Comments {
   userEmail   String?       @db.VarChar(255)
   updatedAt   DateTime?     @db.DateTime(0)
   approvedBy  Int?
+  approver    users?        @relation("Comments_approvedByTousers", fields: [approvedBy], references: [employee_id], onDelete: NoAction, onUpdate: NoAction, map: "Comments_approved_by_employeeId")
+  user        users?        @relation("Comments_userEmailTousers", fields: [userEmail], references: [email], onDelete: NoAction, onUpdate: NoAction, map: "Comments_created_by_employeeId")
```

Comment:
Only nitpick would be using PascalCase for Prisma relation fields, to keep consistency across the models (or any case but I think Aldo and I have used PascalCase for others 😅 ). Unfortunately I don't think we can add this as a Eslint rule but we can add maybe to wiki as Code Style?

Reply:
It is good idea to add this rule in the wiki for future references. 
And about the relations names I already changed.

Reply:
<img width="719" alt="Screen Shot 2022-10-20 at 9 55 38" src="https://user-images.githubusercontent.com/97203617/196983997-334e953a-e16a-41f6-849c-a7dca25e6d65.png">

Reply:
Thank you, and added #46 ticket 👍